### DISCUSSION TITLE
Question: How to add hover effect on each cell separately?

### DISCUSSION THREAD

**Discussion by samuk190** [namespace-ee/react-calendar-timeline#988] [Q&A]
![image](https://github.com/namespace-ee/react-calendar-timeline/assets/19878001/7cd66cb9-6d71-4d50-b9e1-161e6cfb374a)
I want something like this it is possible?

**Comment by positiveabhii** [comment]
can i try working on this?

**Comment by niranjan821** [comment]
You can use itemRenderer and use mouse events to detect mouse hover.

eg: 
`itemRenderer = ({ item, timelineContext, itemContext, getItemProps, getResizeProps }) => {
    const { left: leftResizeProps, right: rightResizeProps } = getResizeProps();
    const backgroundColor = itemContext.selected ? (itemContext.dragging ? "red" : item.selectedBgColor) : item.bgColor;
    const borderColor = itemContext.resizing ? "red" : item.color;
    return (
      <div
        {...getItemProps({
          style: {
            backgroundColor: 'green',
            color: item.color,
            borderColor,
            borderStyle: "solid",
            borderWidth: 1,
            borderRadius: 4,
            borderLeftWidth: itemContext.selected ? 3 : 1,
            borderRightWidth: itemContext.selected ? 3 : 1
          },
          onMouseDown: () => {
            console.log("on item click", item);
          },
        })}
        onMouseEnter ={ () => {
          console.log('Hovering on item ->', item);
        }}
      >
        {itemContext.useResizeHandle ? <div {...leftResizeProps} /> : null}

        <div
          style={{
            height: itemContext.dimensions.height,
            overflow: "hidden",
            paddingLeft: 3,
            textOverflow: "ellipsis",
            whiteSpace: "nowrap"
          }}
        >
          {itemContext.title}
        </div>

        {itemContext.useResizeHandle ? <div {...rightResizeProps} /> : null}
      </div>
    );
  };`
